Item/tests/Item-specs.js

```diff
@@ -1,147 +1,106 @@
-import {shallow, mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render} from '@testing-library/react';
 
 import {ItemBase} from '../Item';
+
 import css from '../Item.module.less';
 
 describe('Item Specs', () => {
 	test('should support adding text as a child', () => {
 		const expected = 'Hello Item';
 
-		const subject = mount(
-			<ItemBase>
-				{expected}
-			</ItemBase>
-		);
-
-		const actual = subject.text();
+		const {getByText} = render(<ItemBase>{expected}</ItemBase>);
+		const actual = getByText(expected).textContent;
 
 		expect(actual).toBe(expected);
 	});
 
 	test('should support adding a `label`', () => {
 		const expected = 'Example Label';
 
-		const subject = mount(
-			<ItemBase label={expected}>
-				Hello Item
-			</ItemBase>
-		);
-
-		const actual = subject.find(`.${css.label}`).first().text();
+		const {getByTestId} = render(<ItemBase data-testid="item" label={expected}>Hello Item</ItemBase>);
+		const actual = getByTestId('item').children.item(1).children.item(0).children.item(1).textContent;
 
 		expect(actual).toBe(expected);
```

Comment:
```suggestion
		test('should support adding a `label`', () => {
		const expected = 'Example Label';

		const {getByText} = render(<ItemBase label={expected}>Hello Item</ItemBase>);
		const label = getByText('Example Label');

		expect(label).toBeInTheDocument();
	});
```
In my opinion, this is easier to read